fedora/jellyfin.spec

```
@@ -23,22 +23,27 @@ Source16:       jellyfin-firewalld.xml
 
 %{?systemd_requires}
 BuildRequires:  systemd
-Requires(pre):  shadow-utils
 BuildRequires:  libcurl-devel, fontconfig-devel, freetype-devel, openssl-devel, glibc-devel, libicu-devel
-Requires:       libcurl, fontconfig, freetype, openssl, glibc libicu
 # Requirements not packaged in main repos
 # COPR @dotnet-sig/dotnet or
 # https://packages.microsoft.com/rhel/7/prod/
 BuildRequires:  dotnet-runtime-3.1, dotnet-sdk-3.1
-# RPMfusion free
-Requires:       ffmpeg
-
+Requires: %{name}-server = %{version}-%{release}, %{name}-web >= 1, %{name}-web < 2
```

```suggestion
Requires: %{name}-server = %{version}-%{release}, %{name}-web >= 10.6, %{name}-web < 10.7
```
to match what we were used to have